modules/openideal_slideshow/templates/slideshow.html.twig

```diff
@@ -1,5 +1,77 @@
+{# The swiper slideshow. #}
 <div class="openideal-slideshow">
-    {% for item in items %}
-        <img src="{{ item.uri.value | image_style('slideshow_large') }}" alt="{{ item._referringItem.alt }}">
-    {% endfor %}
+    <ul class="swiper-wrapper my-gallery" itemscope itemtype="http://schema.org/ImageGallery">
+        {% for key, item in items %}
+            <li id="{{ key + 1 }}" class="swiper-slide" itemprop="associatedMedia" itemscope
```

Comment:
What IDs will we have in this case? 1, 2, 3?
Can we add some labels to make them more meaningful? (i.e. `slide-N`)?

Reply:
Done.